tests/app/test_jinja_filters.py

```diff
@@ -17,6 +17,7 @@
     map_list_collector_config,
     RadioConfig,
     CheckboxConfig,
+    get_formatted_address,
```

Comment:
These imports should be sorted

Reply:
Again we need to add a linter for things like this (if possible) if we are to enforce. But I have update anyway 👍

Reply:
Thanks - I am in the process of adding a card to integrate `isort` to our linting and make it compatible with Black